File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/helpdesk/shared/employeeinfo.model.ts
import {
    NgForm,
    FormGroup,
    FormControl,
    Validators,
    FormBuilder,
    ReactiveFormsModule
} from '@angular/forms'
import * as moment from 'moment/moment';

export class HlpDskEmployeeInfo {
    //public EmployeeId : number = 0;
    public EmployeeName :string = null;
    public Designation :string = null;
    public DepartmentName :string = null;
    public ContactNumber: string = null;
    public Extension: string = null;
    public SpeedDial: string = null;
    public OfficeHour: string = null;
    public RoomNumber: string = null;
}

File: Code/Components/DanpheEMR.ServerModel/HelpdeskModels/EmployeeInfoModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel.HelpdeskModels
{//For Employee Information 
    public class EmployeeInfoModel
    {
        //NOTE: REMOVE THIS MODEL altogether and create dynamic reports for helpdesk
        [Key]
        public string EmployeeName { get; set; }
        public string Designation { get; set; }
        public string DepartmentName { get; set; }
        public string ContactNumber { get; set; }
        public Int16? Extension { get; set; }
        public Int16? SpeedDial { get; set; }
        public string OfficeHour { get; set; }
        public string RoomNumber { get; set; }//added:sud-16aug--
    }
}
